profiles/coreos/base/package.accept_keywords

```diff
@@ -42,6 +42,7 @@
 # Required for some CVEs
 =app-editors/vim-8.2.5066-r1 ~amd64 ~arm64
 =app-editors/vim-core-8.2.5066-r1 ~amd64 ~arm64
+=sys-libs/zlib-1.2.12-r3 ~amd64 ~arm64
```

Comment:
If you updated the sys-libs/zlib package in portage-stable to latest commit from gentoo, then the keywords wouldn't be needed - zlib 1.2.12-r3 is already stable in Gentoo.